packages/pwa-devdocs/src/venia-pwa-concept/design/global-style/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Global style guide
+---
+
+{: .bs-callout .bs-callout-info}
+**Note:**
+This design specification is still in development.
+If have any feedback or would like to join the PWA conversation, please join our [Slack][] channel.
+
+This topic provides the global style rules and values used in the Venia theme.
+Use this guide to style components that will fit within the Venia theme.
+
+## Colors
+
+{: .bs-callout .bs-callout-tip}
+***Tip:** Color values are expressed in RGB format because it translates easier to `rgba()` values than HEX format.*
+
+The Venia theme uses the following color classes in its design:
+
+| CSS class         | RGB value                     | Usage            |
+| ----------------- | ----------------------------- | ---------------- |
+| `black`           | `rgb(0, 0, 0)`                |                  |
```

Comment:
`black` might be a global style, but unlike all of these others, it's not a custom property. I'm not sure I'd put it in this list.